lambdas/es/indexer/index.py

```diff
@@ -230,27 +230,32 @@ def handler(event, context):
                 # Consume and ignore this event, which is an initial message from
                 # SQS; see https://forums.aws.amazon.com/thread.jspa?threadID=84331
                 continue
-            else:
-                print("Unexpected message['body']. No 'Records' key.", message)
-                raise Exception("Unexpected message['body']. No 'Records' key.")
+            print("Unexpected message['body']. No 'Records' key.", message)
+            raise Exception("Unexpected message['body']. No 'Records' key.")
         batch_processor = DocumentQueue(context)
         events = body_message.get("Records", [])
         s3_client = make_s3_client()
         # event is a single S3 event
         for event_ in events:
             try:
                 event_name = event_["eventName"]
-                # only process these two event types
-                if event_name not in [OBJECT_DELETE, OBJECT_PUT]:
+                # Process all Create:* and Remove:* events except for delete markers,
+                # which have no contents, don't signify an actual delete, and don't
+                # belong in the index as documents
+                if (event_name == "ObjectRemoved:DeleteMarkerCreated"
+                        or not any(event_name.startswith(n) for n in EVENT_PREFIX.values())):
+                    # TODO: delete this print
+                    print(f"index.py skipping event {event_name}", event_)
                     continue
+                # TODO: delete this print
```

Comment:
That print looks useful to leave in the lambda logs.

Reply:
Doesn't that just slow things done and add to the logging bill though? Think of yuge buckets.